app/upgrades/v8/feegrant.go

```
@@ -0,0 +1,76 @@
+		return err != nil
+	})
+	if iterErr != nil {
+		return iterErr
+	}
+	if err != nil {
+		return err
+	}
+	return err
+}
+
+func swapAllowance(any *types1.Any) (*types1.Any, error) {
```

golangci-lint (1.62.2)

[warning] redefines-builtin-id: redefinition of the built-in type any

(revive)